data/core/doc/init.lua

```diff
@@ -499,6 +545,36 @@ function Doc:indent_text(unindent, line1, col1, line2, col2)
   return line1, col1 + #text, line1, col1 + #text
 end
 
+
+function Doc:update_max_line_len_range(start_line, end_line)
+  local line_numbers = self.long_lines.line_numbers
+  local max_length = self.long_lines.length
+  end_line = math.min(end_line, #self.lines)
+
+  for line=start_line,end_line do
+    local line_len = #self.lines[line]
+    if line_len >= max_length then
+      max_length = line_len
+      line_numbers[line] = line_len
+    else
+      if line_numbers[line] then line_numbers[line] = nil end
+    end
+  end
+  for n, len in pairs(line_numbers) do
+    line_numbers[n] = len >= max_length and len or nil
+  end
+  if not next(line_numbers) then
+    -- Recalc needed
+    self.long_lines.length = 0
+    self.long_lines.line_numbers = line_numbers
```

Comment:
This assignment does nothing I think because the lh side was already equal to the rh side but that's ok.